lib/Model/index.ts

```diff
@@ -132,7 +132,7 @@ async function updateTimeToLive(model: Model<DocumentCarrier>): Promise<void> {
 		break;
 	}
 }
-function waitForActive(model: Model<DocumentCarrier>, forceRefreshOnFirstAttempt = true) {
+function waitForActive(model: Model<DocumentCarrier,any>, forceRefreshOnFirstAttempt = true) {
```

Comment:
Wish this could again be enforced as an error in the linter haha.

```suggestion
function waitForActive(model: Model<DocumentCarrier, any>, forceRefreshOnFirstAttempt = true) {
```